### ISSUE TITLE
forkJoin not completing when one of observables is loadFromDelayedObservable delayType = 'none'

### ISSUE THREAD

**Issue by jaybloke** [Nodonisko/ionic-cache#78]
Not sure if this a bug or simply can't be achieved when subscribing to multiple cache observables with forkJoin.

Example:

```
plan(): Observable<any> {
  let request = this.http.get<any>(`${ENV.apiHost}/plan`).map(response => response.plan);
  return this.cache.loadFromObservable('plan', request, 'group', 60000);
}

services(): Observable<any> {
  let request = this.http.get<any>(`${ENV.apiHost}/services`).map(response => response.services);
  return this.cache.loadFromDelayedObservable('services', request, 'group', 60000, 'none');
}

Observable.forkJoin([this.plan(), this.services()]).subscribe(
  (results: any[]) => {
    console.log('Not called once services in cache');
  }
)
```

Basically, once services is in the cache, the call to forkJoin never completes.

**Comment by darthdie** [comment]
Thanks. I believe I know what the issue is. I'll confirm a fix in the next day or so.

**Comment by jaybloke** [comment]
Nice one @darthdie ! Highly appreciate your quick response 👍

**Comment by darthdie** [comment]
Just published version 3.1.2 with a fix for this. Let me know if you have any more issues, thanks!

**Comment by jaybloke** [comment]
Just tried the new version out and happy to say it works like a dream - Thanks @darthdie for fixing this so quickly!
